src/main/java/org/broadinstitute/hellbender/tools/FixCallSetSampleOrdering.java

```diff
@@ -45,7 +45,7 @@
     @Argument(fullName = GenomicsDBImport.SAMPLE_NAME_MAP_LONG_NAME,
```

Comment:
Oh dear, I didn't realize this tool was still in the GATK -- it should really be removed....

Reply:
I went to remove it but GenomicsDB depends on code in the tools class, so instead I made a [ticket](https://github.com/broadinstitute/gatk/issues/6849).